### PR TITLE
Made the XML layout of Leave form page #10 

### LINKED ISSUE #10: XML Layout for Leave Form Activity
Create the XML layout for Leave Form Activity based on the design provided in the [Figma](https://www.figma.com/design/jYm7XGidk1L3HNf9e4Otwn/myLeave?node-id=6-18&t=HjrOtyAfoLbxQDPn-1).

Your layout must follow these guidelines:

1. Use Constraint Layout to ensure the layout is responsive to all screen sizes.
2. Use Scroll View to add scrolling feature in the form.
3. Adhere to the given Figma design
4. Follow the specified naming convention for each view group ID to maintain consistency and clarity.

### PR DESCRIPTION
**Pull request by AyushNarsian**
1. This PR fixes #10 
2. This PR does the following:
The XML layout of Leave form page has been made along with minor fixes in the code. Also made 2 drop down menus for department box and leave type box;

## Essential Checklist

- [ x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x ] "Allow edits from maintainers" is checked. (See [here](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork) for instructions on how to enable it.)
- [ x] The PR is made from a branch that's **not** called "main/master".

## Proof that changes are correct
[
Screenshot 2024-06-12 015058
](url)

## PR Pointers

- If you need a review or an answer to a question, and don't have permissions to assign people, **leave a comment** like the following: "{{Question/comment}} @{{reviewer_username}} PTAL".
- Never force push. If you do, your PR will be closed.

### CODE REVIEW

File: app/src/main/res/layout/activity_leave_form.xml [hunk 1]
```diff
@@ -7,4 +7,297 @@
...
+                android:layout_height="wrap_content"
+                android:gravity="center"
+                android:fontFamily="@font/epilogue_bold"
+                android:textSize="23sp"
+                android:textFontWeight="600"
+                android:text="@string/apply_for_leave"
+                android:textAppearance="?android:attr/textAppearanceLarge"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toTopOf="parent" />
```
Comment (Agrim2112) [inline app/src/main/res/layout/activity_leave_form.xml:36]: Give proper ID to all views according to the convention, you can take reference from [here](https://stackoverflow.com/questions/12870537/android-naming-convention)

File: app/src/main/res/layout/activity_leave_form.xml [hunk 2]
```diff
@@ -7,4 +7,297 @@
...
+
+                    <EditText
+                        android:id="@+id/editTextName"
+                        android:layout_width="0dp"
+                        android:layout_height="48dp"
+                        android:fontFamily="@font/interr"
+                        android:layout_marginTop="4dp"
+                        android:hint="Agrim Gupta"
+                        android:inputType="textPersonName"
+                        android:textSize="16sp"
```
Comment (Agrim2112) [inline app/src/main/res/layout/activity_leave_form.xml:72]: Increase the top margin for all Edit Text views and also do not give predefined heights, try using wrap_content and give enough padding.

**Review by Agrim2112 — CHANGES_REQUESTED** [review]
Make the requested changes.

**Comment by AyushNarsian** [comment]
Made the requested changes, also added comments for making the code more readable

**Comment by Agrim2112** [comment]
@AyushNarsian Good work.

**Pull request merged**